src/lightning_lite/strategies/launchers/xla.py

```diff
@@ -86,8 +86,7 @@ def _wrapping_function(
         return_queue: SimpleQueue,
         global_states: Optional[_GlobalStateSnapshot] = None,
     ) -> None:
-        # TODO(lite): Update worker setup once TPUSpawn strategy is in Lite
-        self._strategy._worker_setup(process_idx)
+        self._strategy._local_rank = process_idx
```

Comment:
@awaelchli I believe this is not correct, as this no longer sets `XLAStrategy._launched=True` and then #14926 fails with

```python
  File "/home/runner/work/lightning/tests/tests_lite/strategies/test_xla.py", line 17, in broadcast_on_tpu_fn
    result = strategy.broadcast(obj)
  File "/home/runner/work/lightning/src/lightning_lite/strategies/xla.py", line 146, in broadcast
    data_tensor = torch.tensor(data, device=self.root_device, dtype=torch.float)
  File "/home/runner/work/lightning/src/lightning_lite/strategies/xla.py", line 72, in root_device
    raise RuntimeError("Accessing the XLA device before processes have spawned is not allowed.")
```

I'm a bit confused about what's the best way to do this. 

Do `strategy.setup_environment()`?

If yes:
  - This does not set the local rank. Should it? or do we still manually set the local rank?
  - Should this also be done for the other Lite launchers?

Also, why did Lite remove `_worker_setup`? The logic being different between PL and Lite strategies is confusing.

This blocks #14926

Reply:
This was done in reaction to your comment: https://github.com/Lightning-AI/lightning/pull/11073#issuecomment-1247355717
I think it is correct, otherwise many many tests for ddp spawn would fail, and tpu spawn is not fundamentally different regarding this local rank business. 

I commented on https://github.com/Lightning-AI/lightning/pull/14926 that maybe all that is missing in the test is a strategy.setup_environment().